WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -365,7 +365,18 @@ struct ReaderPostMenuButtonTitles {
     }
 
     class func dispatchToggleFollowSiteMessage(post: ReaderPost, follow: Bool, success: Bool) {
-        dispatchToggleFollowSiteMessage(siteTitle: post.blogNameForDisplay(), siteID: post.siteID, follow: follow, success: success)
+        let blogName = {
+            guard let blogNameForDisplay = post.blogNameForDisplay() else {
+                if let siteID = post.siteID, let postID = post.postID {
+                    WordPressAppDelegate.crashLogging?.logMessage("Expected blogNameForDisplay() to exist",
```

Comment:
@dvdchr The optional `WordPressAppDelegate.crashLogging` gives me the impression that if for unknown reasons, that instance is `nil`, that won't message be logged. What do you think of using the non-optional `CrashLogging.main` instance?

Reply:
Good catch! The `main` instance provides a fallback in case the one from the app delegate is nil. This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20567/commits/84b32045bae812841e3136b4526cde2b1c236b90 ! Thanks.

Reply:
Whoops, looks like I missed an import. Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20567/commits/9e320b70cac245870b208f3f84322f2569e42590.